content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object.rb

```diff
@@ -18,6 +25,20 @@ def self.log_and_exit(msg, exit_code, handle = $evm)
               handle.log('info', "Script ending #{msg} code : #{exit_code}")
               exit(exit_code)
             end
+
+            # If you want to create a notification and log a message without specifying a handle using the global $evm
+            #    ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log_and_notify(:level, msg, subject)
```

Comment:
@billfitzgerald0120 Do you want to change the comment to say :notify_level instead of level